src/com/hiuni/milkwars/events/ClanKillCounterEvent.java

```diff
@@ -51,11 +51,11 @@ public static void onPlayerDeath (PlayerDeathEvent event) {
             return;
         }
 
-        event.setDeathMessage(Announce.formatPlayerName(player, friendlyClan) + " of the " +
-                friendlyClan.getName() + " clan died at the hands of " +
-                Announce.formatPlayerName(killer, enemyClan) + " and the " +
-                enemyClan.getName() + " clan.");
-        enemyClan.addKill();
```

Comment:
You commented out the addkill method lol

Reply:
whoops